libs/medic/runner.js

```diff
@@ -35,6 +35,11 @@ const replicateAll = async (couchUrl, pgconn, opts) => {
     if (opts.syncLogsDb) {
       await replicateDatabase(`${couchUrl}-logs`, 'couchdb_medic_logs');
     }
+
+    if (opts.syncUsersDb) {
+      const couchUrlWithoutPath = couchUrl.substring(0, couchUrl.lastIndexOf('/'));
```

Comment:
I dislike all url manipulation in this file. I think the URL never even gets normalized. Not something that needs to be done in this PR though.